Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/ReflectionAdapter.h

```diff
@@ -60,6 +60,9 @@ namespace AZ::DocumentPropertyEditor
         //! property editor instances has altered its value.
         void NotifyPropertyChanged(const PropertyChangeInfo& changeInfo);
 
+        void* GetInstance() { return m_instance; }
+        AZ::TypeId GetTypeId() { return m_typeId; }
```

Comment:
The GetTypeId function could be const.
There can also be as second overload of `GetInstance()` that returns a `const void*` that is also a const member.

Reply:
done.